src/components/routes/homepage/EventForm.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Container,
+  FormControl,
+  InputLabel,
+  Input,
+  TextField,
+  Typography,
+  makeStyles
+} from '@material-ui/core';
+import { EventTimePicker } from './EventTimePicker';
+import { ImagePicker } from './ImagePicker';
+import { MediaCard } from './MediaCard';
+import { Modal } from './Modal';
+import { validateEvent } from '../../../functions/validateEvent';
+import imagePlaceholder from '../../../assets/no_image_placeholder.png';
+
+const useStyles = makeStyles(theme => ({
+  formContainer: {
+    minHeight: '100vh',
+    paddingTop: theme.spacing(10),
+    textAlign: 'center',
+    display: 'grid',
+    gridTemplateColumns: 'repeat(2, 50%)',
+    alignItems: 'center',
+    justifyContent: 'center',
+    [theme.breakpoints.down(800)]: {
+      minHeight: 'max-content',
+      display: 'unset'
+    }
+  },
+  input: {
+    margin: `${theme.spacing(2)}px 0px`
+  },
+  timepickersContainer: {
+    margin: theme.spacing(2),
+    padding: `${theme.spacing(1)}px 0px`,
+    display: 'grid',
+    gridTemplateColumns: '45% 10% 45%'
+  },
+  submitButton: {
+    margin: theme.spacing(5),
+    gridColumn: '1 / span 2',
+    justifySelf: 'center',
+    color: '#fff',
+    backgroundColor: theme.palette.primary.light,
+    '&:hover': {
+      backgroundColor: theme.palette.primary.main
+    }
```

Comment:
pretty sure material-ui `Button`s have a default implementation for this behaviour. is this customised?

Reply:
yeahh I tried without this, but when the buttom is hovered, it gives you default color like white or blue (from `color: primary` or something like that). But let's see if we can get rid of this and still achieve consistent color scheme

Reply:
see if `theme` configuration allows for button customisation

Reply:
alrightt 👍